synapse/storage/databases/main/receipts.py

```diff
@@ -170,8 +170,8 @@ def get_last_receipt_for_user_txn(
         receipt_types: Collection[str],
     ) -> Optional[Tuple[str, int]]:
         """
-        Fetch the event ID and stream_ordering for the latest receipt in a room
-        with one of the given receipt types.
+        Fetch the event ID and stream_ordering for the latest unthreaded receipt
+        in a room with one of the given receipt types.
```

Comment:
If it's possible to get a threaded receipt, seems like we should rename this function, `get_last_unthreaded_receipt_for_user_txn`